Python/CMakeLists.txt

```diff
@@ -171,6 +171,14 @@ set(PRP_AVTR_HDRS
     PRP/Avatar/pySittingModifier.h
 )
 
+set(PRP_CAMERA_SRCS
```

Comment:
The [CMakeLists.txt for the `core` library](https://github.com/Lunanne/libhsplasma/blob/master/core/CMakeLists.txt#L155) uses **CMRA** for this.  I think we should probably use it here too for consistency.